docs/get-started/deploy-bosh-using-arm-templates.md

```diff
@@ -73,18 +59,3 @@ Run the following commands in your home directory to deploy bosh:
 
 * Never use root to perform these steps.
 * More verbose logs are written to `~/run.log`.
```

Comment:
To ensure users do not get stuck when they see errors, we should include a troubleshooting section within the deployment document. In the future, this troubleshooting section should grow as more issues get reported. For example, we could include this new issue #55 in this section as well.